forms.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.8.2/css/bulma.css"
+    />
+    <link rel="stylesheet" href="form.css" />
+    <script src="http://code.jquery.com/jquery-3.5.1.js"></script>

```

Comment:
Don't keep commented code in the PRs. Please remove it

Reply:
ok anna